bin/run-dev.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
 ./bin/run-common.sh
-./manage.py runsslserver 0.0.0.0:8000
+./manage.py runserver 0.0.0.0:8000
```

Comment:
IIRC Persona requires using SSL to login, have you confirmed Persona still working after this change?